src/constructors/injectGlobal.js

```diff
@@ -5,6 +5,10 @@ import type { Interpolation, Stringifier } from '../types'
 
 export default (stringifyRules: Stringifier, css: Function) => {
   const injectGlobal = (strings: Array<string>, ...interpolations: Array<Interpolation>) => {
+    if (process.env.NODE_ENV !== 'production' && !(Array.isArray(strings))) {
+      // eslint-disable-next-line no-console
+      console.warn('Incorrect using injectGlobal as a function. Instead use it as a tagged template literal, see: https://www.styled-components.com/docs/api')
```

Comment:
I think using `console.error` is more correct, since it's not deprecation or something like deoptimization.